MathMagician/Program.cs

```diff
@@ -3,13 +3,69 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using MathMagician.Numbers;
 
 namespace MathMagician
 {
     class Program
     {
         static void Main(string[] args)
         {
+        GETCOMMAND:
+            Console.WriteLine("What would you like me to Do?");
+            Console.WriteLine("Hint: Options are natural, odd, even, fibonacci, prime.");
+            // "How" you provide the user useful info is on you.
+            // Think about, 'How will teh user pick a command?'
+            // Is it better to modify the original question? Make it more useful?
+            string command = Console.ReadLine();
+            // Once I have the command (whatever it looks like), how do I check if
+            // it's a valid command?
+            // when should you actually create an instance of your number class? now or later?
+            NaturalNumber myNumber;
+            switch (command.ToLower())
+            {
+                case "natural":
+                    myNumber = new NaturalNumber();
+                    break;
+                case "odd":
+                    myNumber = new Odd();
+                    break;
+                case "even":
+                    myNumber = new Even();
+                    break;
+                case "fibonacci":
+                    myNumber = new FibonacciNumber();
+                    break;
+                case "prime":
+                    myNumber = new PrimeNumber();
+                    break;
+                default:
+                    Console.WriteLine("Sorry, wrong command!");
+                    goto GETCOMMAND;
```

Comment:
Good use of `goto`. Are you an ol' Skool programmer or know any? I'm interested is learning what let you to this solution.